The WindowX6\4 CI failed with
```
1700850812986 ********************
_bk;t=1700850812987 UNRESOLVED: Flang :: Lower/OpenMP/declare-target-implicit-func-and-subr-cap.f90 (849 of 2420)
_bk;t=1700850812987 ******************** TEST 'Flang :: Lower/OpenMP/declare-target-implicit-func-and-subr-cap.f90' FAILED ********************
_bk;t=1700850812987 Exception during script execution:
_bk;t=1700850812987 Traceback (most recent call last):
_bk;t=1700850812987   File "C:\ws\src\llvm\utils\lit\lit\worker.py", line 76, in _execute_test_handle_errors
_bk;t=1700850812987     result = test.config.test_format.execute(test, lit_config)
_bk;t=1700850812987   File "C:\ws\src\llvm\utils\lit\lit\formats\shtest.py", line 29, in execute
_bk;t=1700850812987     return lit.TestRunner.executeShTest(
_bk;t=1700850812987   File "C:\ws\src\llvm\utils\lit\lit\TestRunner.py", line 2243, in executeShTest
_bk;t=1700850812987     parsed = parseIntegratedTestScript(test, require_script=not script)
_bk;t=1700850812987   File "C:\ws\src\llvm\utils\lit\lit\TestRunner.py", line 2105, in parseIntegratedTestScript
_bk;t=1700850812987     parsed = _parseKeywords(
_bk;t=1700850812987   File "C:\ws\src\llvm\utils\lit\lit\TestRunner.py", line 2044, in _parseKeywords
_bk;t=1700850812987     for line_number, command_type, ln in parseIntegratedTestScriptCommands(
_bk;t=1700850812987   File "C:\ws\src\llvm\utils\lit\lit\TestRunner.py", line 1280, in parseIntegratedTestScriptCommands
_bk;t=1700850812987     f = open(source_path, "rb")
_bk;t=1700850812987 OSError: [Errno 22] Invalid argument: 'C:\\ws\\src\\flang\\test\\Lower\\OpenMP\\declare-target-implicit-func-and-subr-cap.f90'
```
I am not sure how it is related to this PR change. I don't have a window machine to reproduce it. 
Is there any way that I can re-run the CI on the agent?

--- a/flang/unittests/Runtime/RuntimeCrashTest.cpp
+++ b/flang/unittests/Runtime/RuntimeCrashTest.cpp
@@ -43,9 +43,9 @@ TEST(TestTerminator, CheckFailedLocationTest) {
 }
 
 TEST(TestTerminator, CheckFailedTest) {
-  static Fortran::runtime::Terminator t;
+  static Fortran::runtime::Terminator t("someFileName");
   ASSERT_DEATH(t.CheckFailed("predicate"),
-      "RUNTIME_CHECK\\(predicate\\) failed at \\(null\\)\\(0\\)");
+      "RUNTIME_CHECK\\(predicate\\) failed at someFileName\\(0\\)");
 }
 
 //------------------------------------------------------------------------------

